client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -65,7 +65,7 @@ const siteSetupFlow: Flow = {
 	useSteps() {
 		return [
 			STEPS.GOALS,
-			STEPS.INTENT,
+			STEPS.INTENT_I2,
```

Comment:
Temporarily changed to easily show changes in this PR within the `/site-setup` flow.